lib/testing_appserver/data/helper_files/helper_files.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module TestingAppServer
+  class HelperFiles
+    PATH_TO_FILE = "#{ENV['HOME']}/RubymineProjects/testing-appserver/lib/testing_appserver/data/helper_files/"
```

Comment:
I think all those Constanct can be replaced to methods, it allow more flexibility in future

so it will be

```
def root_path
#{ENV['HOME']}/RubymineProjects/testing-appserver/lib/testing_appserver/data/helper_files/"
end

def xlsx_name
'Schülerwettbewerb der Ingenieur ёñ.xlsx'
end
```

And all can be wrapped in 

```
self << 
```
So there is no instance class of `HelperFiles`
and can be accessed by `HelperFiles.xlsx_name`

Reply:
`Tempfile` is not working because api method returns `"message": "Empty file"` while file uploading to portal

Reply:
Maybe its better to divide `shared_example/documents_filter/docx_xlsx_pptx_users_groups_filter.rb` into:
- docx_xlsx_pptx_filter
- users_groups_filter

because now there is rubocop offense `Metrics/ParameterLists` for too match params